controllers/controllers.js

```diff
@@ -0,0 +1,15 @@
+const { selectTopics } = require('../models/models');
+
+const getTopics = (req, res, next) => {
+  return selectTopics()
+    .then((rows) => {
+      res.status(200).send({ topics: rows });
+    })
+    .catch(next);
+};
+
+const sendBadRequest = (req, res, next) => {
+  return Promise.reject({ status: 400, msg: 'Bad Request' }).catch(next);
+};
```

Comment:
This is a bit unusual, not sure about the utility of having a controller that just rejects a promise - wherever you need to, you can make use of Promise.reject -  but I'm not sure I see the reason for it being in it's own model function. Was this just for debugging / testing? If so then make sure to remove it once you're done debugging (just like console.logs)

Reply:
This function is for when there is an unknown endpoint such as '/banana'. I could have put it in app.js but thought it would be cleaner to move it here. I will now move it back to app.js